packages/client/src/services/CredentialTypeService.ts

```diff
@@ -0,0 +1,62 @@
+// src/services/CredentialTypeService.ts
+
+import { CredentialTypeInfo, ImportCredentialTypeOptions } from '@2060.io/model'
+import { Logger } from 'tslog'
+
+import { ApiVersion } from '../types/enums'
+
+const logger = new Logger()
+
+/**
+ * `CredentialTypeService` class for managing credential types and interacting with
+ * the available endpoints related to credential types in the Agent Service.
+ *
+ * This class provides methods for querying, creating, and managing credential types.
+ * For a list of available endpoints and functionality, refer to the methods within this class.
+ */
+export class CredentialTypeService {
+  private url: string
+
+  constructor(
+    private baseURL: string,
+    private version: ApiVersion,
+  ) {
+    this.url = `${this.baseURL.replace(/\/$/, '')}/${this.version}/credential-types`
+  }
+
+  public async importCredentialType(importData: ImportCredentialTypeOptions): Promise<CredentialTypeInfo> {
```

Comment:
To make the API calls shorter, given that that these methods are already scoped we can skip the "CredentialType" word from the names. I.e.:

- client.credentialTypes.import()
- client.credentialTypes.create()
- client.credentialTypes.getAll()